kafka-ui-api/src/main/java/com/provectus/kafka/ui/rest/KafkaConnectRestController.java

```diff
@@ -0,0 +1,92 @@
+package com.provectus.kafka.ui.rest;
+
+import com.provectus.kafka.ui.api.ApiClustersConnectApi;
+import com.provectus.kafka.ui.cluster.service.KafkaConnectService;
+import com.provectus.kafka.ui.model.*;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ServerWebExchange;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import javax.validation.Valid;
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+@Log4j2
+public class KafkaConnectRestController implements ApiClustersConnectApi {
+    private final KafkaConnectService kafkaConnectService;
+
+    @Override
+    public Mono<ResponseEntity<Flux<Connect>>> getConnects(String clusterName, ServerWebExchange exchange) {
+        return kafkaConnectService.getConnects(clusterName).map(ResponseEntity::ok);
+    }
+
+    @Override
+    public Mono<ResponseEntity<Flux<String>>> getConnectors(String clusterName, String connectName, ServerWebExchange exchange) {
+        Flux<String> connectors = kafkaConnectService.getConnectors(clusterName, connectName);
+        return Mono.just(ResponseEntity.ok(connectors));
+    }
+
+    @Override
+    public Mono<ResponseEntity<Connector>> createConnector(String clusterName, String connectName, @Valid Mono<NewConnector> connector, ServerWebExchange exchange) {
```

Comment:
Do we need to wrap the response in ResponseEntity, because it's always OK in your code? as I remember we can tell open API generator do not wrap the response into ResponseEntity

Reply:
We use `ResponseEntity` classes in other parts of the code

Reply:
I see, but the purpose of using this is not clear for me...